src/libraries/Microsoft.Extensions.DependencyInjection/tests/DI.Tests/ServiceCollectionDescriptorExtensionsTests.cs

```diff
@@ -5,7 +5,6 @@
 using Microsoft.Extensions.DependencyInjection.Extensions;
 using Microsoft.Extensions.DependencyInjection.Specification.Fakes;
 using Xunit;
-using Xunit.Abstractions;
```

Comment:
(nit) I'd revert this, if you need to reset CI. No need to change unrelated files and mess up source history.